package.json

```diff
@@ -6,24 +6,20 @@
     "#build-utils": "./build-utils.js"
```

Comment:
Intentional diff here?

Reply:
If you mean the whole file then yes.
Added this:
```
"build:cpp-wasm-cmake-test": "cd fbw-common/src/wasm/cpp-msfs-framework/test && ./test-cpp.sh",
```
and file autoformat (json) removed the empty lines which is why I extended the divider lines to find stuff easier. 

The line you marked does not have any changes, does it?

Reply:
Yes I meant the whole file, thanks for the response.